raddb/debug.conf

```diff
@@ -14,6 +14,31 @@
 #  packaging system. It is only for developers.
 #
 
+#
+#  debug_level:: The base log level for the server.
+#
+#debug_level = 0
+
+#
+#  line_number:: Enable log src file and line number.
+#
+#line_number = false
+
+#
+#  talloc_memory_limit:: Limit the amount of talloced memory the server uses.
+#
+#talloc_memory_limit = 10485760
+
+#
+#  talloc_memory_report:: Print a summary report of all memory used.
+#
+#talloc_memory_report = false
+
+#
+#  talloc_pool_size:: A talloc pool is a pure optimization for specific situations.
+#
+#talloc_pool_size = 8192
```

Comment:
These are all developer only options and should not be documented in the public config (well except debug_level).  If you want to add a "do nothing" flag (just expands to 0) like FR_TYPE_HIDDEN to the CONF_PARSER entries, and then ignore lines containing that, that's probably the best way to manage this.